cypress/integration/unit_tests/checkbox_test.js

```diff
@@ -15,52 +15,52 @@ const colorProperties = {
 };
 const mockFirebaseLayers = [
   {
-    'ee-name': 'asset0',
-    'asset-type': LayerType.FEATURE_COLLECTION,
-    'display-name': 'asset0',
-    'display-on-load': true,
-    'color-function': colorProperties,
+    'eeName': 'asset0',
```

Comment:
nit: I forgot to do this in my PR, but we can actually get rid of the quotes around the properties now in these object initializations.